src/bare-metal/aps/examples/src/uart.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ANCHOR: Example
+use core::ptr::write_volatile;
+
+/// Minimal driver for an 8250 UART. This only implements enough to work with the emulated 8250
```

Comment:
I suggest wrapping the comments at less than 80 characters to avoid clipping:

![image](https://user-images.githubusercontent.com/89623/222727400-a1f743ae-e2b3-423f-bb55-e841bd44cd66.png)

The `rustfmt.toml` file in the repository has `max_width = 90` but that might even be too generous (and `rustfmt` doesn't wrap comments out of the box).

Reply:
Done (in #482).